src/cljs/org/broadinstitute/firecloud_ui/endpoints.cljs

```diff
@@ -100,8 +100,8 @@
       {:path (str "/workspaces/" (ws-path workspace-id) "/checkBucketReadAccess")
        :method :get})
 
-(defn update-workspace-acl [workspace-id]
-  {:path (str "/workspaces/" (ws-path workspace-id) "/acl")
+(defn update-workspace-acl [workspace-id invite-new?]
+  {:path (str "/workspaces/" (ws-path workspace-id) "/acl?inviteUsersNotFound=" (some? invite-new?))
```

Comment:
This is weird.  If you pass `false`, `(some? false)` is `true`.  I think you can just replace `some?` with `true?`.

You could also use the "optional map arguments" pattern, which you can see in utils/ajax-orch.  It would look something like:

    (defn update-workspace-acl [workspace-id & {:keys [invite-new?] :or {invite-new? false}}]
      ...)

You would then call it like this:

    (endpoints/update-workspace-acl my-workspace-id :invite-new? true)